From e9c0d63b22bcb4c31cfaac00fc1271713c8bd3b3 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 8 Aug 2024 10:56:16 +0100
Subject: [PATCH] docs: Restate position on removal of VMware Tools

With the "reboot after every step" change made in the previous commit,
it seems removal of VMware Tools is now more reliable.  Update the
notes about this.

Updates: commit c952f310c902e438a8b0b5240a4b486b698bede8
---
 docs/virt-v2v.pod | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/virt-v2v.pod b/docs/virt-v2v.pod
index 587b4604..346be606 100644
--- a/docs/virt-v2v.pod
+++ b/docs/virt-v2v.pod
@@ -1308,9 +1308,8 @@ Virt-v2v attempts to remove VMware Tools.  For Windows guests this is
 supposed to happen during the first boot after conversion.
 
 We use VMware's recommended uninstallation method as that is the
-safest choice.  However our experience has shown that this method
-usually fails.  If so, VMware Tools must be removed by some other
-method.
+safest choice.  If this fails, VMware Tools must be manually removed
+by some other method.
 
 One possible method is described here:
 L<https://gist.github.com/broestls/f872872a00acee2fca02017160840624>
